examples/aya-tool/myapp-ebpf/rust-toolchain.toml

```diff
@@ -1,2 +1,2 @@
 [toolchain]
-channel="nightly-2023-01-10"
+channel = "nightly-2023-01-10"
```

Comment:
We don't pin nightly anymore. Are you sure you used the latest aya-template?

Reply:
unpinned, also in `.github/workflows/ci.yml`.  "sync with template" is figuratively speaking in the sense to reduce the differences I noted - I don't know a way to "mechanically" sync the two.